### PR TITLE
Deletes the clown outfit from the game.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
A relic from a past age that nobody uses except for unfunny LRP jokes. It's time to step into the modern age.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

**Review by Geevies — APPROVED** [review]
this really goes against the soul of aurorastation (froked from bystation)

File: code/modules/mining/abandonedcrates.dm [hunk 1]
```diff
@@ -118,15 +118,6 @@
...
 			new /obj/item/dnainjector/xraymut(src) // Probably the least OP
-		if(94) // Why the hell not
-			new /obj/item/storage/backpack/clown(src)
-			new /obj/item/clothing/under/rank/clown(src)
-			new /obj/item/clothing/shoes/clown_shoes(src)
-			new /obj/item/modular_computer/handheld/pda/civilian/clown(src)
-			new /obj/item/clothing/mask/gas/clown_hat(src)
-			new /obj/item/bikehorn(src)
-			new /obj/item/pen/crayon/rainbow(src)
-			new /obj/item/reagent_containers/spray/waterflower(src)
```
Comment (fernerr) [inline code/modules/mining/abandonedcrates.dm:129]: This needs something to replace it as loot.

Comment (NonQueueingMatt) [reply code/modules/mining/abandonedcrates.dm:129]: I hate Clowns

**Review by TheDanseMacabre — APPROVED** [review]
Poggers Champion

**Review by JohnWildkins — DISMISSED** [review]
I cannot, in good faith, let a PR like this go through. Such a thing would go against every coding standard I've ever been raised to.






You gotta replace the loot! Perhaps `/obj/random/voidsuit` or something of the like.

**Comment by Stalkeros2** [comment]
No.

**Pull request merged**